packages/plugins/response-cache-cloudflare-kv/src/index.ts

```diff
@@ -44,36 +47,71 @@ export function createKvCache(config: KvCacheConfig): Cache {
   }
   const computedTtlInSeconds = Math.max(Math.floor((config.cacheReadTTL ?? 60000) / 1000), 60); // KV TTL must be at least 60 seconds
 
-  const cache: Cache = {
-    async get(id: string) {
-      const kvResponse = await config.KV.get(buildOperationKey(id, config.keyPrefix), {
-        type: 'text',
-        cacheTtl: computedTtlInSeconds,
-      });
-      if (kvResponse) {
-        return JSON.parse(kvResponse) as ExecutionResult;
-      }
-      return undefined;
-    },
+  return function KVCacheFactory(ctx: TServerContext): Cache {
```

Comment:
We can probably even warn only once at the start of the function isn't it ?

```suggestion
  return function KVCacheFactory(ctx: TServerContext): Cache {
    if (!ctx[config.KVName]) {
      // eslint-disable-next-line no-console
      console.warn(
        `Cloudflare KV namespace ${config.KVName} is not available in the server context, cache will be skipped.`,
      );
      return;
    }
```

Reply:
I don't know the lifecycle of CF Workers, so I thought it might be added later on after startup. I preferred to check it lazily.